.github/workflows/CI.yml

```diff
@@ -40,4 +40,4 @@ jobs:
           files: lcov.info
       - uses: julia-actions/julia-uploadcoveralls@v1
         env:
-          COVERALLS_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
+          COVERALLS_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
```

Comment:
It seems this change solve the issues, right?

Reply:
Indeed!